crates/rethnet_evm/tests/blockchain.rs

```diff
@@ -0,0 +1,368 @@
+use std::str::FromStr;
+
+use rethnet_eth::{
+    block::{Block, DetailedBlock, PartialHeader},
+    transaction::{EIP155TransactionRequest, SignedTransaction, TransactionKind},
+    Address, Bytes, B256, U256,
+};
+use rethnet_evm::{
+    blockchain::{BlockchainError, LocalBlockchain, SyncBlockchain},
+    state::HybridState,
+    SpecId,
+};
+
+async fn create_dummy_blockchains() -> Vec<Box<dyn SyncBlockchain<BlockchainError>>> {
+    const DEFAULT_GAS_LIMIT: u64 = 0xffffffffffffff;
+    const DEFAULT_INITIAL_BASE_FEE: u64 = 1000000000;
+
+    let state = HybridState::default();
+
+    let local_blockchain = LocalBlockchain::new(
+        &state,
+        U256::from(1),
+        SpecId::LATEST,
+        U256::from(DEFAULT_GAS_LIMIT),
+        None,
+        Some(B256::zero()),
+        Some(U256::from(DEFAULT_INITIAL_BASE_FEE)),
+    )
+    .expect("Should construct without issues");
+
+    // TODO: enable forked blockchain once hashing of transactions works properly
```

Comment:
Is there a tracking issue for this? Would be good to add it to the comment

Reply:
Good catch. I should have re-enabled that myself as I fixed the issue. Addressed in https://github.com/NomicFoundation/hardhat/pull/4300/commits/808055957cf39bf59186efe1e57f92a1e2d490e1